synapsis/src/main/java/com/soen/synapsis/appuser/jobposting/JobPosting.java

```diff
@@ -0,0 +1,90 @@
+package com.soen.synapsis.appuser.jobposting;
+
+import com.soen.synapsis.appuser.AppUser;
+import com.soen.synapsis.appuser.Role;
+
+public class JobPosting {
```

Comment:
Is this class meant to represent a data model? If so, it should be marked with @Entity, and the attributes given appropriate annotations as well for JPA to work (see AppUser as an example of what I mean).